src/sso.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
     CONFIG,
 };
 
-pub static FAKE_IDENTIFIER: &str = "Vaultwarden";
+pub static FAKE_IDENTIFIER: &str = "DUMMY_IDENTIFIER_FOR_OIDC";
```

Comment:
```suggestion
pub static FAKE_IDENTIFIER: &str = "VW_DUMMY_IDENTIFIER_FOR_OIDC";
```

I know it is not necessary, but I usually like to use project specific namespaces for stuff like that. But if you are that concerned, a uuidv4 should also do it. ;-)

Reply:
I have added the prefix, I prefer something which make it clear that it's a dummy value as opposed to a more opaque uuid.